tests/test_connection_options.py

```diff
@@ -6,6 +6,7 @@
 import pytest
 
 from databases.backends.aiopg import AiopgBackend
+from databases.backends.mysql import MySQLBackend
```

Comment:
How was this working before?

Reply:
This was coming from the import. Although this works with any version anyway. No worries, rolling back now